neuro_san/client/agent_cli.py

```diff
@@ -124,7 +118,17 @@ def chat(self):
 3. Is the value for the agent name key in the server manifest.hocon file set to true?
 4. Servers will skip manifest entries that have errors. They will also print out which
    agents they are actually serving.  Check your server output for each of these.
+5. Is the server itself actually running?
 """
```

Comment:
Define the message earlier as now it might be used in more than one place.
Add a new checklist item.